src/wix/test/WixToolsetTest.CoreIntegration/TestData/Comments/Default.wxs

```diff
@@ -0,0 +1,9 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <Package Version="1"
+           Name="MsiPackage"
+           Manufacturer="Example Corporation"
+           UpgradeCode="047730a5-30fe-4a62-a520-da9381b8226a">
+    <MajorUpgrade DowngradeErrorMessage="Downgrade not allowed" />
+    <SummaryInformation />
```

Comment:
A `SummaryInformation` without attributes shouldn't be allowed, but not necessary to fix it as part of this change.